master/getting-started/kubernetes/reqs/system.md

```diff
@@ -5,35 +5,44 @@ canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/r
 
 {% include {{page.version}}/reqs-sys.md orch="Kubernetes" %}
 
-## Kubernetes
+## Kubernetes requirements
 
 #### Supported versions
 
-{{site.prodname}} {{page.version}} requires Kubernetes 1.8 or later.
+We test against the following Kubernetes versions.
```

Comment:
I think it's nice to have the version here, to make it extra clear this applies to a single release stream of Calico.